src/audio/midi/MidiAudio.cpp

```diff
@@ -20,11 +19,16 @@ constexpr auto WAVE_DATA = std::array<byte, WAVE_SIZE>{ 128, 159, 191, 223, 255,
 
 MidiAudio::MidiAudio(HWND windowHandle)
 {
-    if (SUCCEEDED(DirectSoundCreate8(nullptr, &_directSound, nullptr)))
+    _directSoundModule.reset(LoadLibraryExW(L"dsound.dll", nullptr, LOAD_LIBRARY_SEARCH_SYSTEM32));

```

Comment:
@j4james Two things!

1. Congrats, `DECPS` is now supported in Windows! Rolling out to an insider build near you in a few weeks probably...
2. It turns out that there are versions of Windows that do not have DirectSound; I hope this is an acceptable solution!

Reply:
Excellent! That looks good to me.